apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -40,6 +40,39 @@
             <tr><th>Instance</th><td colspan="2"><%= assessment_instance.number %>
                 (<a href="<%= plainUrlPrefix %>/course_instance/<%= course_instance.id %>/assessment_instance/<%= assessment_instance.id %>">student view</a>)
             </td></tr>
+            <tr>
+              <th>Fingerprint Changes</th>
+              <td colspan="2">
+                <%= assessment_instance.client_fingerprint_id_change_count %>
+                <% if (assessment_instance.client_fingerprint_id_change_count > 0) { %>
+                  <div tabindex="0" class="btn btn-xs" role="button"
+                      data-toggle="popover" data-trigger="focus" data-container="body"
+                      data-html="true" title="Client Fingerprint ID Changes"
+                      data-content="<table class=&quot;table&quot;>
+                        <tr>
+                          <th>Event</th>
+                          <th>Time</th>
+                          <th>Fingerprint ID</th>
+                          <th>IP address</th>
+                          <th>Session ID</th>
+                        </tr>
+                          <% log.forEach(function(row) { %>
+                            <% if (row.client_fingerprint_id) { %>
+                              <tr>
+                                <td><%= row.event_name %></td>
+                                <td><%= row.formatted_date %></td>
+                                <td><%= row.client_fingerprint_id %></td>
+                                <td><%= row.ip_address %></td>
+                                <td><%= row.session_id %></td>
+                              </tr>
+                            <%  } %>
+                          <%  }) %>
```

Comment:
This popover window is quite narrow so the table is overflowing onto the page. Is there a way to widen this popover window?

Reply:
Consider using a full modal for this instead of just a popover: https://getbootstrap.com/docs/4.6/components/modal/

Reply:
https://github.com/PrairieLearn/PrairieLearn/blob/2007eb47919523c469ae510f85560ac2784fc4ee/apps/prairielearn/assets/scripts/lib/questionsTable.js#L173-L175

Reply:
I have updated this to use a modal instead of a popover in commit b135adf. I think provides a better user experience. 
![Screenshot 2023-11-08 at 10 49 53 AM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/8ce5aa62-d216-4121-865a-669b24bf371d)